vars/karateTestUtils.groovy

```diff
@@ -181,14 +181,17 @@ void createFailedFeatureJiraIssue(KarateModuleExecutionSummary moduleSummary, Ka
         Labels     : [KarateConstants.ISSUE_LABEL]
     ]
 
-    def teamName = "TEAM_MISSING"
-    def team = teamByModule[moduleSummary.name]
-    if (team) {
-        teamName = team.name
-        fields["Development Team"] = teamName
-    } else {
-        echo "Module ${moduleSummary.name} is not assigned to any team."
-    }
+    def teamName = "Kitfox"
+    fields["Development Team"] = teamName
+
+//    def teamName = "TEAM_MISSING"
+//    def team = teamByModule[moduleSummary.name]
+//    if (team) {
+//        teamName = team.name
+//        fields["Development Team"] = teamName
```

Comment:
you're in Git world...
no need to preserve commented lines.

Reply:
I was left up to the next PR. It's going to send Jira Tickets to Kitfox for 2-3 days. It will be uncommented after that period.